app/src/main/java/com/aragh/kotlin2/screen/users/UsersPresenter.kt

```diff
@@ -0,0 +1,31 @@
+package com.aragh.kotlin2.screen.users
+
+import com.aragh.kotlin2.interactor.Users
+import com.aragh.kotlin2.screen.CoroutinePresenter
+import kotlinx.coroutines.experimental.android.UI
+import kotlin.coroutines.experimental.CoroutineContext
+
+
+class UsersPresenter(private val users: Users,
+                     coroutineContext: CoroutineContext = UI)
+  : CoroutinePresenter(coroutineContext), PresenterContract {
+
+  override var view: ViewContract? = null
+    set(value) {
+      field = value
+      value?.showUsers(emptyList())
+    }
+
+
+  override fun onStart() {
+    runInCoroutine(
+        { users.getAllUsers() },
+        { view?.showUsers(it) },
+        {}
```

Comment:
Redundant it seems Yoda, maybe you could introduce default params as `= {}` to avoid code like this?